manifests/mod/fcgid.pp

```diff
@@ -1,17 +1,24 @@
 class apache::mod::fcgid(
   $options = {},
 ) {
+  if $::osfamily == 'RedHat' and $::operatingsystemmajrelease == '7' {
```

Comment:
Was this your original goal? Because #1178 doesn't talk about FreeBSD.

If you goal is just FreeBSD, add it to the condition instead of breaking every non-Redhat distro.